scapy/asn1/ber.py

```diff
@@ -256,7 +256,7 @@ def safedec(cls, s, context=None):
 
     @classmethod
     def enc(cls, s):
-        if type(s) is str:
+        if isinstance(s, str):
```

Comment:
Maybe we should use `basestring` here (that we will replace using `six.string_types`).